lib/naughty/core.lua

```diff
@@ -591,8 +591,7 @@ function naughty.notify(args)
     local destroy_cb = args.destroy
 
     -- beautiful
-    local beautiful = bt.get()
-    local font = args.font or preset.font or
+    local font = args.font or preset.font or beautiful.notification_font or
```

Comment:
Since the commit message doesn't mention it and I do not see an added `@beautiful`: Why does this add a new theme variable?